README.md

```diff
@@ -42,9 +42,11 @@ As a starting point for your own project take a look at https://github.com/ether
 
 ##### Running from command line:
 ```
-> git clone https://github.com/ethereum/ethereumj
-> cd ethereumj
-> ./gradlew run [-PmainClass=<sample class>]
```

Comment:
@Nashatyrev I’ve created a PR without the dependency updates for `develop` #376. Same code regarding README.md. 

> ./gradlew run [-PmainClass=<sample class>]
…is not gone as below there are several examples using this pattern.